pages/playlist/[id].js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import viewPlaylistDetails from '../../api/mergedData';
+import PlaylistForm from '../../components/forms/PlaylistForm';
+
+export default function ViewPlaylists() {
+  const [editItem, setEditItem] = useState({});
+  const router = useRouter();
+
+  const { id } = router.query;
+
+  useEffect(() => {
+    viewPlaylistDetails(id).then(setEditItem);
+  }, [id]);
+
+  return (
+    <PlaylistForm playlistObj={editItem} />
```

Comment:
We should be returning the single playlist card with the associated podcast cards, not the playlist form. :)

Reply:
We fixed this last night